recipe/meta.yaml

```
@@ -40,7 +41,7 @@ requirements:
   run:
     - python
     - joblib >=1.2.0
-    - {{ pin_compatible('numpy') }}
+    - numpy
```

```suggestion
    - numpy >=1.19.5
```
https://github.com/scikit-learn/scikit-learn/blob/1.5.2/pyproject.toml#L10C9-L10C17